Add an option to disable corba no matter how orocos-rtt is configured

--- config/FindOrocosRTT.cmake~	2008-05-09 11:58:58.000000000 +0200
+++ config/FindOrocosRTT.cmake	2011-10-05 15:46:11.582460663 +0200
@@ -70,6 +70,7 @@
 	  #ENDIF (CORBA_ENABLED)
 	ENDIF ( OROCOS_TARGET STREQUAL "${FLAVOR_TARGET}")
 	
+if(CORBA_ENABLED)
 	IF(OROCOS_CORBA_TARGET STREQUAL "${FLAVOR_TARGET}")
 	  MESSAGE( "Detected CORBA build for ${OROCOS_TARGET}" )
 	  PKGCONFIG("orocos-rtt-${FLAVOR_TARGET}" CORBA_ENABLED  OROCOS_RTT_CORBA_INCLUDE_DIRS OROCOS_RTT_CORBA_DEFINES OROCOS_RTT_CORBA_LINK_DIRS OROCOS_RTT_CORBA_LIBS )
@@ -83,6 +84,7 @@
 	  ADD_DEFINITIONS( ${OROCOS_RTT_CORBA_DEFINES} )
 	  
 	ENDIF(OROCOS_CORBA_TARGET STREQUAL "${FLAVOR_TARGET}")
+endif(CORBA_ENABLED)
       ENDFOREACH(FLAVOR ${OROCOS_RTT_FLAVORS})
       
       IF( NOT OROCOS_RTT )
